src/bin/zmcontactbackup

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/perl
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2018 Synacor, Inc.
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+
+=head1 NAME
+
+zmcontactbackup - Start/Stop/Schedule contact backup on mailbox server(s)
+
+=head1 SYNOPSIS
+
+zmcontactbackup <[-h|-m|-r|-p|-q|-d|-f]|[[-a|--append] {<backup-schedule>}]> [list of mailbox servers]
```

Comment:
nit: this should probably be:
``` zmcontactbackup <-h|-m|-r|-p|-q|-d|-f|[-a backup_schedule]> [mailbox_server...]```

Trying to base this off of this reference: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html

Reply:
done